### ISSUE TITLE
Why there are no input normalization in mobilenet version?

### ISSUE THREAD

**Issue by JC1DA** [ildoonet/tf-pose-estimation#369]
As far as I know, mobilenet requires inception preprocessing to [-1, 1] range but I didn't see any normalization for the input? Any problems with that?

**Comment by ariel415el** [comment]
Hi,
I want to join your question and add that none of my training on coco with mobilenet_thin architecture had converged.
In addition all training losses are in significantly larger than 1 and are still ~500 for even after 15 epochs

Was somebody able to train succsessfully?

**Comment by ildoonet** [comment]
@JC1DA @ariel415el Thanks. I will train with the normalization(preprocessing to -1 to +1). 

@ariel415el losses are less than 80~100 after 15 epoch for my case. Try with other hyperparameters.

**Comment by mgarbade** [comment]
As far as I can see it, the  "Mobilenet V2" with pretrained imagenet weights, expect the following preprocessing:

- RGB (currently the channel order is BGR due to the usage of opencv)
- normalization [0..255]  -> [-1..1] (this is currently only implemented for the CMU model but not for the mobilenet versions)

Changing the channel order could be done using 
```
im = cv2.imread(img_filename)
im = cv2.cvtColor(im, cv2.COLOR_BGR2RGB)
```
